app/views/layouts/_google_analytics_script.haml

```diff
@@ -75,19 +33,9 @@
       if (typeof _gaq != 'undefined') {
         _gaq.push(['_trackEvent'].concat(params_array));
       }
-  - if feature_enabled?(:customer_universal_analytics)
-    :plain
-      if (typeof ga_customer === 'function'){
-        ga_customer('send', 'event', category, action, opt_label)
-      }
-  - else
-    :plain
-      var params_array = [category, action, opt_label]
-      if (typeof _gaq != 'undefined') {
-        if (typeof secondary_analytics_in_use != 'undefined' && secondary_analytics_in_use) {
-          _gaq.push(['b._trackEvent'].concat(params_array));
-        }
-      }
+  :plain
+    if (typeof customer_report_event === 'function') {
```

Comment:
If you are refactoring this more, you could add some comments in these if-elses. So that it would be more clear which analytics contribute to these and removing legacy + features would be easier.